components/job-orchestration/job_orchestration/reducer/reducer.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import subprocess
+import sys
+from pathlib import Path
+from typing import List
+
+from clp_py_utils.clp_config import CLPConfig
+from clp_py_utils.clp_logging import get_logger, get_logging_formatter, set_logging_level
+from clp_py_utils.core import read_yaml_config_file
+from pydantic import ValidationError
+
+logger = get_logger("reducer")
+
+
+def main(argv: List[str]) -> int:
+    args_parser = argparse.ArgumentParser(description="Spin up reducers.")
+    args_parser.add_argument("--config", "-c", required=True, help="CLP configuration file.")
+    args_parser.add_argument("--host", required=True, help="Host that the reducers should bind to")
+    args_parser.add_argument(
+        "--concurrency", required=True, help="Number of reducer servers to run"
+    )
+    args_parser.add_argument(
+        "--polling-interval-ms", required=True, help="Database polling interval in ms"
```

Comment:
This isn't really the "Database polling interval" since the reducer doesn't talk to the metadata database anymore, right? This is more of the upsert timer interval or words to that effect?

Reply:
Yep, can do a pass to rename things appropriately